app/content/tiles/jadeCityLight.ts

```diff
@@ -176,25 +178,25 @@ const jadeCityLightEdges: TileSource = {
     w: 16, h: 16,
     source: requireFrame('gfx/tiles/jadeCityLight.png', {x: 0, y: 32, w: 80, h: 64}),
     behaviors: {
-        'all': { defaultLayer: 'field' },
-        '0x0': { defaultLayer, solidMap: BITMAP_LEFT_6, ledges: {left: true}},
-        '1x0': { defaultLayer, solidMap: BITMAP_LEFT_6, ledges: {left: true}},
-        '3x0': { defaultLayer, solidMap: BITMAP_RIGHT_6, ledges: {left: true}},
-        '4x0': { defaultLayer, solidMap: BITMAP_RIGHT_6, ledges: {left: true}},
-        '0x1': { defaultLayer, solidMap: orBitMasks(BITMAP_TOP_6, BITMAP_LEFT_6), ledges: { left: true, up: true } },
-        '1x1': { defaultLayer, solidMap: BITMAP_TOP_6, ledges: { up: true } },
-        '2x1': { defaultLayer, solidMap: BITMAP_TOP_6, ledges: { up: true } },
-        '3x1': { defaultLayer, solidMap: BITMAP_TOP_6, ledges: { up: true } },
-        '4x1': { defaultLayer, solidMap: orBitMasks(BITMAP_TOP_6, BITMAP_RIGHT_6), ledges: { right: true, up: true } },
-        '0x2': { defaultLayer, solidMap: BITMAP_LEFT_6, ledges: { left: true } },
-        '1x2': { defaultLayer, solidMap: BITMAP_TOP_6, ledges: { up: true } },
-        '3x2': { defaultLayer, solidMap: BITMAP_TOP_6, ledges: { up: true } },
-        '4x2': { defaultLayer, solidMap: BITMAP_RIGHT_6, ledges: { right: true } },
-        '0x3': { defaultLayer, solidMap: orBitMasks(BITMAP_BOTTOM_10, BITMAP_LEFT_6), ledges: { left: true, down: true } },
-        '1x3': { defaultLayer, solidMap: BITMAP_BOTTOM_10, ledges: { down: true } }, //if not approaching from the bottom, consider versions that use foreground2
-        '2x3': { defaultLayer, solidMap: BITMAP_BOTTOM_10, ledges: { down: true } }, //so PC can be covered by them.
-        '3x3': { defaultLayer, solidMap: BITMAP_BOTTOM_10, ledges: { down: true } },
-        '4x3': { defaultLayer, solidMap: orBitMasks(BITMAP_BOTTOM_10, BITMAP_RIGHT_6), ledges: { right: true, down: true } },
+        'all': { defaultLayer: 'field', linkedOffset: 159 },
```

Comment:
This is probably my mistake, but it looks like you can just use `defaultLayer,` here instead of `defaultLayer: 'field',`.